guides/hhvm/02-basic-usage/03-server.md

```diff
@@ -1,46 +1,48 @@
-Server mode is how you will use HHVM to serve web requests. The HHVM process starts up and continuously waits to serve web requests.
+Use HHVM server mode to create a HHVM process that continuously serves web requests, with these advantages:
 
-Multiple requests can, of course, be served simultaneously, and HHVM also caches code to be shared across requests as well.
+✔️ Multiple requests served simultaneously
```

Comment:
Could we use plain bullet points here? I'm not sure if ticks will look good on all browsers.

Reply:
Sure. Was trying to get fancy, but didn't think of that as a concern.